MapboxVision/Services/Recording/RecordCoordinator.swift

```diff
@@ -49,7 +49,6 @@ final class RecordCoordinator {
     
     private let videoRecorder: VideoBuffer
     private let videoTrimmer: VideoTrimmer
```

Comment:
`VideoTrimmer` also needs updated settings. We may call `trim` method passing settings so we don't need to store them inside.